sig-instrumentation/annual-report-2021.md

```diff
@@ -0,0 +1,74 @@
+*   What programs do you participate in for new contributors?
+    *   Individual/1-1 mentoring
+    *   We try to feature “good-first-issue”s and raise them on k-dev (e.g. structured logging)
+*   Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
+    *   Yes: Google, Red Hat, and many first contributors across multiple companies
+        *   Cross referencing [PRs by contributor + company](https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20quarter&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All) with our [members](https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-instrumentation/teams.yaml#L20), we have 6 companies with contributions: Google, Red Hat, Polar Signals, Huawei, Salesforce, and ByteDance.  The vast (80%) majority of our contributions from members in the past quarter are from Google and Red Hat.
+        *   In [only sig-instrumentation-owned repositories](https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20quarter&var-metric=contributions&var-repogroup_name=SIG%20Instrumentation&var-country_name=All&var-companies=All), there are 18 contributors with >10 contributions in the last quarter, from 14 different companies.
+
+## Current initiatives and project health
+
+*   What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?
+    *   [Structured Logging](https://github.com/kubernetes/enhancements/issues/1602)
+    *   [Tracing](https://github.com/kubernetes/enhancements/issues/647)
+    *   [Reducing metrics exposed by the kubelet](https://github.com/kubernetes/kubernetes/issues/68522?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDM3ODU4MzI4NTo0NzA2MTMx&notifications_query=is%3Asaved#issuecomment-715462010)
+*   Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+    *   [Structured logging Alpha in 1.19](https://github.com/kubernetes/enhancements/issues/1602)
+        *   Flag to enable structured logs: --logging-format=json
+        *   For beta: perf tests, continued migration, verify decisions through one complete component migrated
+    *   [Defend Against Logging Secrets via Static Analysis: Alpha in 1.20](https://github.com/kubernetes/enhancements/issues/1933)
+        *   Test static analysis against sampled PRs using prow
+    *   [Pod Resource Metrics Alpha in 1.20](https://github.com/kubernetes/enhancements/issues/1748)
+        *   Pod Resource Metrics representing the view of the scheduler
+    *   [Log sanitization Alpha 1.20](https://github.com/kubernetes/enhancements/issues/1753)
+        *   Allow filtering out known sensitive data via flag --logging-sanitization
+    *   Next:
```

Comment:
Not sure if kube-state-metrics 2.0 efforts should be mentioned here? @brancz ?

Reply:
yes!

Reply:
Maybe:
```
* cut the release of major breaking v2.0.0 of kube-state-metrics 
```

Reply:
Should this really be mentioned under the `Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?` part?

Maybe we could add a whole new bullet point for the SIG sub projects and add a few words about kube-state-metrics there?

Reply:
Yes this was just a random point I added the comment, agreed its up to @ehashman where to place it.